SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -599,6 +599,7 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
     if (Coord_i[1] < EPS) return;
 
     AD::SetPreaccIn(Coord_i[1]);
+    AD::SetPreaccIn(V_i[2]);
```

Comment:
Thank you for the fix, this is probably something I broke recently.
Can you add a simple regression test case? It can be a pipe or something very small, just enough to prevent this in the future.

There is an issue opened for the SA problem you mention #1565, we should check for this in CConfig::SetPostprocessing and throw an error (if axisymetry and the turbulence model is not NONE, SST, or SST_SUST).